pennylane/resource/specs.py

```diff
@@ -272,22 +511,167 @@ def circuit():
                 qml.RandomLayers(qml.numpy.array([[1.0, 2.0]]), wires=(0, 1))
                 return qml.expval(H)
 
-        >>> from pprint import pprint
-        >>> pprint(qml.specs(circuit, level="user")())
-        CircuitSpecs(device_name='default.qubit',
-                     num_device_wires=None,
-                     shots=Shots(total_shots=None, shot_vector=()),
-                     level='user',
-                     resources=[SpecsResources(gate_types={'RandomLayers': 1},
-                                               gate_sizes={2: 1},
-                                               measurements={'expval(Prod(num_wires=2, num_terms=2))': 1},
-                                               num_allocs=2,
-                                               depth=1),
-                                SpecsResources(gate_types={'RandomLayers': 1},
-                                               gate_sizes={2: 1},
-                                               measurements={'expval(Prod(num_wires=2, num_terms=2))': 1},
-                                               num_allocs=3,
-                                               depth=1)])
+        ::
+
+            >>> from pprint import pprint
+            >>> pprint(qml.specs(circuit, level="user")())
+            CircuitSpecs(device_name='default.qubit',
+                         num_device_wires=None,
+                         shots=Shots(total_shots=None, shot_vector=()),
+                         level='user',
+                         resources=[SpecsResources(gate_types={'RandomLayers': 1},
+                                                   gate_sizes={2: 1},
+                                                   measurements={'expval(Prod(num_wires=2, num_terms=2))': 1},
+                                                   num_allocs=2,
+                                                   depth=1),
+                                    SpecsResources(gate_types={'RandomLayers': 1},
+                                                   gate_sizes={2: 1},
+                                                   measurements={'expval(Prod(num_wires=2, num_terms=2))': 1},
+                                                   num_allocs=3,
+                                                   depth=1)])
+
+    .. details::
+        :title: Using specs on workflows compiled with Catalyst
+
+        The available options for `levels` are different for circuits which have been compiled using Catalyst.
```

Comment:
```suggestion
        The available options for ``levels`` are different for circuits which have been compiled using Catalyst.
```

So annoying :old-man-yells-at-sphinx: